### PR TITLE
add print statement

### PR DESCRIPTION
**Pull request by isobelsnellenberger**
Isobel say hi to Jane

### CODE REVIEW

File: snellenberger2.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+print("Isobel says 'Hello Jane' :+)")
```
Comment (mwcraig) [inline snellenberger2.py]: I really think the space should be after the colon, not before, so ` :+)` → `: +)`.

Comment (isobelsnellenberger) [reply snellenberger2.py]: no

Comment (mwcraig) [reply snellenberger2.py]: Oh, right, I misunderstood the emoji. Kids these days!

I would an an exclamation point though, so maybe `'Hello Jane!'`. That way the sentence is a real sentence.

**Review by mwcraig — CHANGES_REQUESTED** [review]

**Review by mwcraig — APPROVED** [review]

**Comment by mwcraig** [comment]
Thanks for the change!

**Pull request merged**